app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,14 @@
+.curious {
+  background: asset-url("curiousity.jpg") no-repeat center center scroll;
+  background-size: cover;
+}
+
+.curious h1 {
+  padding: 3rem;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.